Sources/StreamChat/Database/DTOs/PollOptionDTO.swift

```diff
@@ -58,7 +58,7 @@ extension PollOptionDTO {
 
 extension PollOptionDTO {
     func asModel() throws -> PollOption {
-        guard !isDeleted else { throw ClientError.DeletedModel(self) }
+        guard !isDeleted else { throw DeletedModel(self) }
```

Comment:
Couldn't add a method to all DTOs, like: `try isNotDeleted()` ?  So that don't repeat this guard everywhere 🤔

Reply:
Good idea